### DISCUSSION TITLE
How To Run The File Meta

### DISCUSSION THREAD

**Discussion by AlyciaTelun** [Dijji/FileMeta#145] [Q&A]
I am using Windows 10 and I am not sure how to run the file. Can someone assist me ?

**Comment by bridgewalker888** [comment]
Hi Alycia.  First, go to this link:  **https://github.com/Dijji/FileMeta/wiki/**Getting-Started****.  Thereafter, go to your Windows 10 start button and click it, (a search window should open).  Type in Windows' "File Association Manager."  Thence, follow the instructions contained in the above link.  Hope this helps ... Aloha.
